dart/lib/flat_buffers.dart

```diff
@@ -138,9 +140,13 @@ class Builder {
   /// automatically grow the array if/as needed.  `internStrings`, if set to
   /// true, will cause [writeString] to pool strings in the buffer so that
   /// identical strings will always use the same offset in tables.
-  Builder({this.initialSize: 1024, bool internStrings = false})
-      : _buf = ByteData(initialSize) {
-    if (internStrings == true) {
+  Builder(
+      {this.initialSize: 1024,
+      bool internStrings = false,
+      Allocator allocator = const DefaultAllocator()})
```

Comment:
nit: this will format a little better with a trailing comma after `const DefaultAllocator(),`

Reply:
Personally I like the current version better but hey, I don't really care.

Added a comma.